app/views/booking_requests/visitors_step.html.erb

```diff
@@ -7,11 +7,6 @@
   <%= render('shared/validation', object: f.object) %>
   <div class="grid-row">
     <div class="column-two-thirds">
-      <div class="form-group">
-        <h3 class="bold-medium">
-          <%= t('.your_details') %>
-        </h3>
-      </div>
       <fieldset>
         <legend class="visuallyhidden">
           <%= t('.your_details') %>
```

Comment:
@chrishanes i'm wondering is this can go as well, since it seems to be here for accessibility purpose? I'm be wrong
In the case it can go than maybe we can get rid of the ".your_details" key?